src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
```

Comment:
The message 'Excavator started its work.' could be more informative. Consider including the type of machine in the message, for example: 'The excavator is now operational.'